app/controllers/application_controller.rb

```diff
@@ -1,4 +1,9 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+  private
+
+  def default_url_options(options = {})
+    { locale: I18n.locale }.merge options
```

Comment:
(参考情報なので修正しなくても OK) `reverse_merge` というメソッドを使うとこんなふうに書けます 💁 デフォルト値を設定するときによく使うメソッドです。`options` を主体にできるので自然なコードにできます。 https://api.rubyonrails.org/classes/Hash.html#method-i-reverse_merge

```suggestion
    options.reverse_merge(locale: I18n.locale)
```

Reply:
せっかくなので教えていただいた設定方法に変更しました。
確かにoptionsから始められると読みやすいです👀✨